contracts/src/systems/hyperstructure/contracts.cairo

```diff
@@ -8,6 +8,8 @@ use s0_eternum::{
 };
 use starknet::ContractAddress;
 
+const LEADERBOARD_REGISTRATION_PERIOD: u64 = 604800; // one week
```

Comment:
can we put this into a config ?

Reply:
should be hardcoded in the contracts so we can't change it, loaf mentioned it should be as permissionless as possible

Reply:
ok